types/algoliasearch/index.d.ts

```diff
@@ -2099,4 +2097,6 @@ interface AlgoliaStatic extends AlgoliasearchInstance {
 
 declare const algoliasearch: AlgoliaStatic;
 
-export = algoliasearch;
+declare module 'algoliasearch' {
```

Comment:
this is the old way of making a browser-accessible module. Can you please try this instead:

```ts
export = algoliasearch;
export as namespace algoliasearch;
```

This should allow you to keep your local `Omit` type, and therefore not have to update all the dependents of algoliasearch.

Reply:
Thanks @sandersn, this works perfectly ❤️